agent/src/test/java/com/teamscale/jacoco/agent/options/AgentOptionsTest.java

```diff
@@ -105,6 +106,18 @@ public void testHttpServerOptionsWithCoverageViaHttp() throws AgentOptionParseEx
 		assertThat(agentOptions.shouldDumpCoverageViaHttp()).isTrue();
 	}
 
+	/** Tests that supplying version info is supported in Testwise mode. */
+	@Test
+	public void testVersionInfosInTestwiseMode() throws AgentOptionParseException {
+		AgentOptions agentOptions = getAgentOptionsParserWithDummyLogger().parse("mode=TESTWISE,class-dir=.," +
+				"http-server-port=8081,teamscale-revision=1234");
+		assertThat(agentOptions.getHttpServerPort()).isEqualTo(8081);
```

Comment:
This does not actually test the version infos 🤔

Reply:
I only wanted to make sure we don't get an exception. But I do assert the correct values now.